prospero-cli/src/main/java/org/wildfly/prospero/cli/RepositoryDefinition.java

```diff
@@ -56,4 +69,29 @@ private static boolean isValidUrl(String text) {
             return false;
         }
     }
+
+    public static URI getAbsoluteFileURI(String repoName) throws ArgumentParsingException, URISyntaxException {
+        File file = new File(getPath(repoName));
+        if (file.exists()) {
+            return file.toURI();
+        } else {
+            throw CliMessages.MESSAGES.invalidFilePath(repoName);
+        }
+    }
+
+    public static String getPath(String repoName) throws URISyntaxException {
+        if (repoName.startsWith("file:")) {
+            URI inputUri = new URI(repoName);
+            if (inputUri.getScheme().equals("file") &&
+                    !inputUri.getSchemeSpecificPart().startsWith("/")) {
+                inputUri = new URI(("file:/" +
```

Comment:
IIUC at this point this is a relative URI. In which case Path.of(inputUri.getSchemeSpecificPart()).toAbsolutePath() should be all you need to get the absolute path

Reply:
Also this already creates the Path from the URI, why does it need to be converted back to URI and then again to Path?